docs/source/azure/examples/cosmosdb-collection-on-off-hours-databases.rst

```diff
@@ -0,0 +1,54 @@
+Cosmos DB Collections - Resize Throughput with On/Off Hours
+===========================================================
+
+With Azure Cosmos DB, you pay for the throughput you provision. Sometimes it's known that Cosmos DB will not be
+utilized during certain hours of the day. To save cost during those times, it's useful for the higher throughput
+collections to be resized down to a lower throughput.
+
+Combining the following filters and actions will allow us to resize and restore the throughput state of Cosmos DB
+collections according to hours of the day:
+
+    Filters:
+
+    * ``onhour``: allows us to filter actions so they execute only during on hours on the parent Cosmos DB Account. The filter can be applied using the ``parent`` filter for the Cosmos DB Account.
+    * ``offhour``: allows us to filter actions so they execute only during off hours. The filter can be applied using the ``parent`` filter for the Cosmos DB Account.
+    * ``offer``: allows us to filter collections with high throughputs (in this example, greater than 800)
+
+    Actions:
+
+    * ``store-throughput-state``: stores the current state of the collections in a tag
+    * ``replace-offer``: resizes collections during off hours (in this example, down to 400)
+    * ``restore-throughput-state``: restores the throughput state of the collections from the tag provided in the ``store-throughput-state`` action
```

Comment:
For some reason `store` and `restore` don't click for me and they were similar enough that I kept mixing the actions up.

Thoughts on `save` and `restore`?  

To me using `offer` instead of `throughput` is also worth considering (unless it is factually incorrect somehow).  To me the terms I am familiar with in this context are `offer` and `RU`.

Reply:
Well offer seems to more than just throughput on it though. But I understand the save and store! Can update.

Reply:
Yeah true.